<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_addmusicwidget</class>
 <widget class="QDialog" name="admin_addmusicwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>131</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>08SeoulHangang</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>08SeoulHangang</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>08SeoulHangang</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TITLE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>08SeoulHangang</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>COMPOSER</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>08SeoulHangang</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SINGER</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>08SeoulHangang</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>GENRE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>08SeoulHangang</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TAG</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>251</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="idEdit">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nameEdit">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="composerEdit">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="singerEdit">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_2">
      <property name="font">
       <font>
        <family>08SeoulHangang</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Choose Genre</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>POP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>K-POP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CLASSIC</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <property name="font">
       <font>
        <family>08SeoulHangang</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Choose TAG</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CALM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>EXCITED</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>GLOMMY</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DANCE</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>226</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="okbtn">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelbtn">
      <property name="text">
       <string>CANCEL</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-2</x>
     <y>20</y>
     <width>601</width>
     <height>59</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>08SeoulHangang</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>♬ ADD MUSIC ♬</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
